decidim-core/db/seeds.rb

```diff
@@ -17,8 +17,8 @@
     table.tr("_", "/").classify.safe_constantize
   end.compact.each(&:reset_column_information)
 
-  smtp_label = ENV["SMTP_FROM_LABEL"] || Faker::Twitter.unique.screen_name
-  smtp_email = ENV["SMTP_FROM_EMAIL"] || Faker::Internet.email
+  smtp_label = ENV.fetch("SMTP_FROM_LABEL", nil) || Faker::Twitter.unique.screen_name
```

Comment:
I think better would be:
```suggestion
  smtp_label = ENV.fetch("SMTP_FROM_LABEL", Faker::Twitter.unique.screen_name)
```